components/atoms/BreadLink.js

```diff
@@ -0,0 +1,30 @@
+import propTypes from "prop-types";
+import { faAngleRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Link from "next/link";
+
+export function BreadLink({ url, text, arrow }) {
```

Comment:
Agree this should be it's own component, but will also needs it's own test

Reply:
my initial approach was to make it a component - but i decided against it as it would overcomplicate a very specific chunk of code that is only used by this particular component.  it would never get used anywhere else by any other component, and is more or less just a function that works within this code. we can chat about it more....